diagnosis_api/routes/disease_diagnosis.py

```diff
@@ -13,39 +13,37 @@
 @app.route("/index")
 @app.route("/")
 def index():
-    return jsonify({'message':'Welcome to disease diagnosis API with pandas'}),200
+    return jsonify(
+        {"message": "Welcome to disease diagnosis API with pandas"}), 200
 
 
 """
     Endpoint for getting diseases by symptoms
 """
 
 
-@app.route("/api/v1/get_disease_by_sypmtoms", methods=['GET'])
-
+@app.route("/api/v1/diseases/search", methods=["GET"])
 def search_disease_by_sypmtoms():
     """
         function to get disease by sypmtoms
     """
 
     args = request.args.to_dict()
-    file_name = args['file_name ']
-    search_params_api = args['search_params']
+    file_name = args["file_name "]
+    search_params_api = args["search_params"]
     search_params = ast.literal_eval(search_params_api)
-    search = Search_disease_by_symptoms(file_name,search_params)
+    search = SearchDiseaseBySymptoms(file_name, search_params)
     result = search.search_disease_by_symptoms()
-    return jsonify({'disease':result}),200
-
-
+    return jsonify({"disease": result}), 200
 
-@app.route("/api/v1/get_disease_details", methods=['GET'])
 
+@app.route("/api/v1/diseases/details", methods=["GET"])
 def get_disease_details():
     """
         function to create a association
```

Comment:
I think that the doctring is misleading in this case because the function says get_disease_details. I would expect that the description should relate to the function.

Reply:
Yeah sure,

Reply:
Thank you for that observation